services/agora/src/components/ui-library/button/ZKButton.ts

```diff
@@ -0,0 +1,18 @@
+import { defineComponent, h } from "vue";
+import { QBtn } from "quasar";
+
+export default defineComponent({
+  setup(props, { slots }) {
+    return () =>
+      h(
+        QBtn,
+        {
+          ...props,
```

Comment:
Make sure this is typesafe. `props` should only be of type QBtnProps, and it should be explicit when calling the ZKButton component, *without* extra work on the caller side